src/index.ts

```diff
@@ -100,23 +100,18 @@ function startApp(appSchema, path: string) {
     )
   }
 
+  if (enableSentry) {
+    Sentry.init({
+      dsn: SENTRY_PRIVATE_DSN,
+    })
+    app.use(Sentry.Handlers.requestHandler())
+  }
+
   app.use(requestIDsAdder)
   if (PRODUCTION_ENV) {
     app.set("trust proxy", true)
   }
 
-  if (enableSentry) {
-    raven
-      .config(SENTRY_PRIVATE_DSN, {
-        ignoreErrors: [
```

Comment:
Will these errors start showing up now? Presumably there was a flood of these (quota), so we wanted to just ignore them.

Reply:
That filter wasn't working, since we were on such an old version.